external/llvm-external-projects/torch-mlir-dialects/test/tmtensor/invalid.mlir

```diff
@@ -67,7 +67,7 @@ func @scatter_mixed_tensor_memref(
 func @scatter_mixed_tensor_memref(
     %update : tensor<?x?xf32>, %indices : tensor<?x1xi32>,
     %original : tensor<?x?xf32>) -> memref<?x?xf32> {
-  // expected-error @+1 {{expected type of `outs` operand #0 'tensor<?x?xf32>' to be same as result type 'memref<?x?xf32>'}}
+  // expected-error @+1 {{op result #0 must be ranked tensor of any type values, but got 'memref<?x?xf32>'}}
```

Comment:
These `test/**/*mlir` changes are from a particular llvm-project I was using. I don't expect this would appear in a PR to upstream (nor will it appear when I move llvm-project a little bit further again).